src/main/java/teammates/common/datatransfer/attributes/StudentProfileAttributes.java

```diff
@@ -30,30 +30,6 @@
     public String pictureKey;
     public Date modifiedDate;
 
-    public StudentProfileAttributes(String googleId, String shortName, String email, String institute,
-                                    String nationality, String gender, String moreInfo, String pictureKey) {
-        this.googleId = googleId;
-        this.shortName = SanitizationHelper.sanitizeName(shortName);
-        this.email = SanitizationHelper.sanitizeEmail(email);
-        this.institute = SanitizationHelper.sanitizeTitle(institute);
-        this.nationality = SanitizationHelper.sanitizeName(nationality);
-        this.gender = gender;
-        this.moreInfo = moreInfo;
-        this.pictureKey = pictureKey;
-    }
-
-    public StudentProfileAttributes(StudentProfile sp) {
-        this.googleId = sp.getGoogleId();
-        this.shortName = sp.getShortName();
-        this.email = sp.getEmail();
-        this.institute = sp.getInstitute();
-        this.nationality = sp.getNationality();
-        this.gender = sp.getGender();
-        this.moreInfo = sp.getMoreInfo().getValue();
-        this.pictureKey = sp.getPictureKey().getKeyString();
-        this.modifiedDate = sp.getModifiedDate();
-    }
-
     public StudentProfileAttributes() {
```

Comment:
Can this be `private`, or is there another use case for default attribute outside the builder's call?